src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -146,6 +146,7 @@ protected static function getBundleDefaultConfig()
                 'enabled' => false,
                 'fallbacks' => array('en'),
                 'logging' => true,
+                'cache' => 'translation.cache.default',
```

Comment:
We should probably also add a test where this is explicitly set to the empty string. And what about testing it with an invalid value?